lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -114,8 +114,12 @@ describe('LMSFilePicker', () => {
     assert.equal(fileList.prop('files'), expectedFiles);
   });
 
-  it('shows the "Try again" prompt after a failed authorization attempt', async () => {
-    fakeApiCall.rejects(new ApiError('Not authorized', {}));
+  it('shows the "Authorize" and "Authorize again" buttons after 2 failed authorization requests', async () => {
+    fakeApiCall.rejects(
+      new ApiError('Not authorized', {
+        /** withouth message */
```

Comment:
```suggestion
        /** withouth message */
```

Should this comment say something like "without `errorMessage` property"?

Reply:
Yes, it should be.